include/envoy/buffer/buffer.h

```diff
@@ -77,6 +77,26 @@ class Instance {
    */
   virtual void add(const Instance& data) PURE;
 
+  /**
+   * Prepend data to the buffer.
+   * @param data supplies the data address.
+   * @param size supplies the data size.
+   */
+  virtual void prepend(const void* data, uint64_t size) PURE;
+
+  /**
+   * Prepend a string to the buffer.
+   * @param data supplies the string to copy.
+   */
+  virtual void prepend(const std::string& data) PURE;
```

Comment:
now we have absl::string_view, can this one combined with the one above as `prepend(absl::string_view data)`?

Reply:
Yeah, that's cleaner. I've made the change.